engine/src/main/java/org/terasology/rendering/nui/ScaleMode.java

```diff
@@ -82,6 +82,29 @@ public Vector2f scaleForRegion(Rect2i region, int actualWidth, int actualHeight)
         public String toString() {
             return "tiled";
         }
+    },
+    Automatic_selector {
+        @Override
+        public Vector2f scaleForRegion(Rect2i region, int actualWidth, int actualHeight) {
+
+                if(actualHeight < 500)
+                {
+                    System.out.println("Hell of a job");
```

Comment:
Debug print statements should not be present in the final code.

Additionally the preferred method for printing is to use our provided logger. Using this logger looks like
```java
LoggerFactory.getLogger(<classNameHere>.class);
```
Standard practice is to place it into a static final field,
```java
private static final Logger logger = LoggerFactory.getLogger(<className>.class);
```